.travis.yml

```diff
@@ -3,4 +3,7 @@ python:
     - "2.7"
 install:
     - pip install requests junit_xml beaker-client mock
-script: python -m unittest discover tests
+    - pip install pylint
+script:
+    - python -m unittest discover tests
+    - pylint tests
```

Comment:
Could you please put the pylint invocation first, as it's a more basic, lower level test?

Reply:
My reasoning was that I'd prefer to see what is the result of the test suite when a PR is opened and if pylint doesn't pass is not something useful for the review. It has to pass if you want to see the PR accepted but it does not contribute too much in the discussions.

Reply:
I would just like to have some logical order, it doesn't matter which. I agree, having it "most important first" or "biggest impact first" makes sense, and will work. Could you perhaps put a comment above, specifying which order the tests should be, so we don't have to discuss this again? You can use `echo` to print it, or just `true` with comment as the argument, which will be ignored.

Reply:
Ah, on the second thought, adding the comment about the order matters little at this moment, please feel free to ignore.

Reply:
Ok